include/glow/Importer/Caffe2ModelLoader.h

```diff
@@ -51,14 +51,20 @@ class Caffe2ModelLoader
   llvm::Error loadWeightsFromNet(caffe2::NetDef &net);
 
   /// Loads an individual weight \p op.
+  /// If Bindings are provided creates Placeholders vs Constants.
   llvm::Error loadWeight(const caffe2::OperatorDef &op);
 
   /// Load the structure of the network from the 'net' file.
-  llvm::Error loadNetwork(caffe2::NetDef &net);
+  /// If Bindings are provided creates Placeholders vs Constants.
```

Comment:
If bindings are provided, weights are created as Placeholders. If not, they are created as Constants and their values are taken from the net weight file.